helm/scylla-operator/values.yaml

```diff
@@ -1,3 +1,6 @@
+# Operator replicas use leader election
+replicas: 2
```

Comment:
Although it's not surprising, please explicitly add a warning comment that setting replicas to a lower value then 2 means it won't be HA and won't react during operator upgrades or disruptions.

Reply:
Thanks for your feedback! I added a comment and added a check before creating pdbs